include/boost/geometry/algorithms/convert.hpp

```diff
@@ -80,7 +80,7 @@ struct point_to_box
         typedef typename coordinate_type<Box>::type coordinate_type;
 
         set<Index, Dimension>(box,
-                boost::numeric_cast<coordinate_type>(get<Dimension>(point)));
+                geometry::detail::numeric_cast<coordinate_type>(get<Dimension>(point)));
```

Comment:
This could be `detail::numeric_cast` or if this was implemented in `util` namspace `util::numeric_cast`.

Reply:
`detail` didn't always work, because we use it a lot on lower levels too.

Reply:
for `util` the `geometry::` prefix was not necessary, it's convenient